cmake/schemes/url_sha1_boost.cmake.in

```diff
@@ -147,3 +148,5 @@ set(HUNTER_PACKAGE_INSTALL_PREFIX "@HUNTER_PACKAGE_INSTALL_PREFIX@")
 set(HUNTER_Boost_VERSION "@HUNTER_Boost_VERSION@")
 
 hunter_install_boost_config()
+
+hunter_copy_license_file("@HUNTER_PACKAGE_SOURCE_DIR@" "@HUNTER_PACKAGE_LICENSE_FILE@")
```

Comment:
Use `hunter_test_string_not_empty("@HUNTER_PACKAGE_LICENSE_FILE@")` to detect if this variable is not set (by mistake). See `# Check preconditions` section (I can't add a comment there).